singleflight/singleflight_test.go

```diff
@@ -45,7 +45,7 @@ func TestDoErr(t *testing.T) {
 		return nil, someErr
 	})
 	if err != someErr {
-		t.Errorf("Do error = %v; want someErr", err, someErr)
+		t.Errorf("Do error = %v; want someErr", err)
```

Comment:
Shouldn't this have been changed to `t.Errorf("Do error = %v; want someErr = %v", err, someErr)` instead?